### PR TITLE
Create rev1.py

### PR DESCRIPTION
**Pull request by idlagit**
add addition function. please review

### CODE REVIEW

File: rev1.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import sys
```
Comment (idlagit) [inline rev1.py:1]: Do we need to import sys?

Comment (idlagit) [reply rev1.py:1]: No we don't but we will need it later

**Review by idlagit — COMMENTED** [review]
ok looks good.

**Comment by Mallampython** [comment]
Looks good

**Review by Mallampython — APPROVED** [review]

**Pull request merged**